src/client/datascience/jupyter/kernels/jupyterKernelService.ts

```diff
@@ -66,7 +66,12 @@ export class JupyterKernelService {
         var specFile: string | undefined = undefined;
 
         // If the spec file doesn't exist or is not defined, we need to register this kernel
-        if (kernel.kind !== 'connectToLiveKernel' && kernel.kernelSpec && kernel.interpreter) {
+        if (
+            kernel.kind !== 'connectToLiveKernel' &&
+            kernel.kind != 'startUsingRemoteKernelSpec' &&
```

Comment:
missing =?

Reply:
Yep. Thanks. Which also prompted me to lookup the difference. ```===``` does not do any type manipulation and just compares the values. ```==``` will convert the values to the same types. So I guess ```===``` is faster.